packages/graphql/src/translate/queryAST/utils/create-cypher-subquery.ts

```diff
@@ -18,35 +18,62 @@
  */
 
 import Cypher from "@neo4j/cypher-builder";
-import type { QueryASTContext } from "../ast/QueryASTContext";
+import { QueryASTContext } from "../ast/QueryASTContext";
 import type { AttributeAdapter } from "../../../schema-model/attribute/model-adapters/AttributeAdapter";
 import type { CypherAnnotation } from "../../../schema-model/annotation/CypherAnnotation";
+import type { Field } from "../ast/fields/Field";
 
 export function createCypherAnnotationSubquery({
     context,
     attribute,
     projectionFields,
+    nestedFields,
 }: {
     context: QueryASTContext;
     attribute: AttributeAdapter;
     projectionFields?: Record<string, string>;
+    nestedFields?: Field[];
 }): Cypher.Clause {
     const cypherAnnotation = attribute.annotations.cypher;
     if (!cypherAnnotation) throw new Error("Missing Cypher Annotation on Cypher field");
 
-    const statementSubquery = getStatementSubquery(context, cypherAnnotation);
-
     const columnName = new Cypher.NamedVariable(cypherAnnotation.columnName);
-    const returnVariable = context.getScopeVariable(attribute.name);
+    const returnVariable = context.getScopeVariable(attribute.name) as Cypher.Node;
```

Comment:
Do we need this casting? If so, maybe move the cast to the lowest level, instead of doing this at the beggining